.nvmrc

```diff
@@ -1 +1 @@
-v16.16.0
+v18.18.0
```

Comment:
To be the [same](https://github.com/getflywheel/flywheel-local/blob/b8650337bf628f6efadb771907aaacadd0c498d2/.nvmrc) as Local Core.